tests/test-scenarios-in-runtime.sh

```diff
@@ -21,6 +21,11 @@ realpath() {
 
 TEST_HOME=`dirname $(realpath "$0")`
 
+SKIP_CONTAINER_POLICY=0
```

Comment:
If we merge this script, it tests host policies by default.
So, let's set the flags for container policies by default.
Then, similar to -EnableHostPolicy, why don't we have some options (arguments) for host policy testing?

Reply:
It doesn't actually test the host policy and scenarios if we don't have the enableHostPolicy flag passed kubearmor. It just skips them as of now.

Reply:
I am going to add one argument called -testHostPolicy to both test-scenarios-in-runtime.sh and test-scenarios-local.sh.